packages/tracking/legacy/class.tracks-client.php

```diff
@@ -38,22 +43,21 @@
 	}
 ```
  */
-
 class Jetpack_Tracks_Client {
 	const PIXEL           = 'https://pixel.wp.com/t.gif';
 	const BROWSER_TYPE    = 'php-agent';
 	const USER_AGENT_SLUG = 'tracks-client';
 	const VERSION         = '0.3';
 
 	/**
-	 * record_event
+	 * Record an event.
 	 *
 	 * @param  mixed $event Event object to send to Tracks. An array will be cast to object. Required.
 	 *                      Properties are included directly in the pixel query string after light validation.
 	 * @return mixed         True on success, WP_Error on failure
 	 */
-	static function record_event( $event ) {
-		if ( ! Jetpack::jetpack_tos_agreed() || ! empty( $_COOKIE['tk_opt-out'] ) ) {
```

Comment:
I am going to walk back on some what I said in https://github.com/Automattic/jetpack/pull/13698#discussion_r333576946, sorry. It seems it was added on purpose in #9156, so maybe it's best if we keep it there?

Other than that it should be good to go.

Reply:
done in 8a14c13